src/tests/smoke/max-matches.smoke.ts

```diff
@@ -0,0 +1,8 @@
+import * as smoke from './smoke';
+
+smoke.suite('Smoke → maxMatches', [
+	{
```

Comment:
Looks like this test does not work as expected. This behaviour related to the comment about wrong place for filtering.

You can add `debug: true` to the test and the console will contain the following lines:

```
  Smoke → maxMatches
FIXTURES                                  NODE_GLOB  FAST_GLOB
----------------------------------------  ---------  ---------
fixtures/file.md                          +          +
fixtures/first                            +          +
fixtures/first/file.md                    +          +
fixtures/first/nested                     +          +
fixtures/first/nested/directory           +          +
fixtures/first/nested/directory/file.md   +          +
fixtures/first/nested/file.md             +          +
fixtures/second                           +          +
fixtures/second/file.md                   +          +
fixtures/second/nested                    +          +
fixtures/second/nested/directory          +          +
fixtures/second/nested/directory/file.md  +          +
fixtures/second/nested/file.md            +          +

    √ pattern: 'fixtures/**/*' (sync) (17ms)
```

Reply:
I wasn't able to run this one - it seemed to error on my machine